### PR TITLE
fix: path to the HvBaseProps in the components

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/core/src/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { theme } from "@hitachivantara/uikit-styles";
 import { AvatarProps as MuiAvatarProps } from "@mui/material/Avatar";
 import clsx from "clsx";
 import { CSSProperties } from "react";
-import { HvBaseProps } from "types";
+import { HvBaseProps } from "../../types";
 import { useImageLoaded } from "../../hooks";
```
Comment (francisco-guilherme) [inline packages/core/src/components/Avatar/Avatar.tsx:7]: here the import should be from alias hooks

File: packages/core/src/components/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { theme } from "@hitachivantara/uikit-styles";
...
 import { decreaseSize } from "../../utils";
```
Comment (francisco-guilherme) [inline packages/core/src/components/Avatar/Avatar.tsx:8]: here the import should be from alias utils

**Review by francisco-guilherme — CHANGES_REQUESTED** [review]

File: packages/core/src/components/Avatar/Avatar.tsx (hunk 1 above)
Comment (plagoa) [reply packages/core/src/components/Avatar/Avatar.tsx]: PR updated.

File: packages/core/src/components/Avatar/Avatar.tsx (hunk 2 above)
Comment (plagoa) [reply packages/core/src/components/Avatar/Avatar.tsx]: PR updated.

**Comment by plagoa** [comment]
@francisco-guilherme updated PR with changes requested.

**Review by francisco-guilherme — APPROVED** [review]

**Pull request merged**